ONSdigital/eq-survey-runner
# Sanitise logging

### Changes

Modifies the logging statements to ensure that no information is leaked when logging errors
### How to test

1) Checkout the branch
2) Set the EQ_LOGGING variable to ERROR
3) Run the app and break what you can
4) Check that no sensitive information is displayed in the logs at ERROR level
### Who can test

Anyone except @weapdiv-david and @LJBabbage
